File: Beecrowd/2455 Gangorra/main.c
#include <stdio.h>

int main(){
    int P1, C1, P2, C2 , esq, dir;

    scanf("%d %d %d %d", &P1, &C1, &P2, &C2);

    esq = P1 * C1;
    dir = P2 * C2;

    if(esq == dir)
        printf("0\n");
    else if(esq > dir)
        printf("-1\n");
    else
        printf("1\n");

    return 0;
}

File: Beecrowd/2455 Gangorra/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int p1, c1, p2, c2, esq, dir;

        p1 = sc.nextInt();
        c1 = sc.nextInt();
        p2 = sc.nextInt();
        c2 = sc.nextInt();

        esq = p1 * c1;
        dir = p2 * c2;

        if (esq == dir)
            System.out.println("0");
        else if (esq > dir)
            System.out.println("-1");
        else
            System.out.println("1");

        sc.close();
    }

}